src/components/contacts/ContactsRemove.js

```diff
@@ -2,62 +2,71 @@ import React from 'react'
-            <Button>REMOVE CONNECTION</Button>
-        </List.Item>
-        <List.Item className='recent-transactions'>
-            <List.Item
-                as='h6'
-                className='recent-transactions-title border-top'
-            >
+
+    <div className='deauthorize-box'>
+        <div className='top contacts'>
+            <div>
+                <div className='title'>
+                    {false &&
+                        <div className='image'>
+                            <MainImage 
+                                src={AccountGreyImage} 
+                                size='medium'
+                            />
+                        </div>
+                    }
+                    <div className='details'>
+                        <h2>
+                            Alex Skidanov
+                        </h2>
+                        <h5>
+                            @alex.near
+                        </h5>
+                    </div>
+                </div>
+            </div>
+            <div className='remove-connection'>
+                <FormButton
+                    className='deauthorize'
+                    color='red'
+                >
+                    REMOVE CONNECTION
+                </FormButton>
+            </div>
+        </div>
+        <div className='recent-transactions'>
+            <h6 className='title border-top'>
                 RECENT TRANSACTIONS
-            </List.Item>
-            <List.Item className='recent-transactions-row border-top'>
-                <List.Header>You sent 20 Ⓝ</List.Header>
-                <List.Item>3h ago</List.Item>
-            </List.Item>
-            <List.Item className='recent-transactions-row border-top'>
-                <List.Header>Alex sent you 1020 Ⓝ</List.Header>
-                <List.Item>3d ago</List.Item>
-            </List.Item>
-            <List.Item className='recent-transactions-row border-top'>
-                <List.Header>
-                    You and Alex played NEAR Chess
-                </List.Header>
-                <List.Item>1w ago</List.Item>
-            </List.Item>
-        </List.Item>
-        <List.Item className='send-money border-top'>
-            <Button as={Link} to='/send-money/marcin'>
-                SEND TOKENS
-            </Button>
-        </List.Item>
-    </List>
+            </h6>
+            <div className='row border-top'>
+                <b className='color-black'>You sent 20 Ⓝ</b>
+                <div>3h ago</div>
+            </div>
+            <div className='row border-top'>
+                <b className='color-black'>Alex sent you 1020 Ⓝ</b>
+                <div>3d ago</div>
+            </div>
+            <div className='row border-top'>
+                <b className='color-black'>You and Alex played NEAR Chess</b>
+                <div>1w ago</div>
+            </div>
+        </div>
+        <div className='send'>
+            <Link to='/send-money/marcin'>
```

Comment:
Minor: Probably better to style `Link` component. Button inside of anchor tag is considered [invalid html5](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element).

Reply:
Yes, good point, this part of code is not visible in the project, but the same problem appears in a few other places. My idea was to use <FormButton> component for all form buttons in the project after that linked button was needed and I used it forgetting about that.

The idea with <FormButton> component is good I think, it will be easy to change all buttons if the design will change, also resolves an issue with repeating css.

My proposition is to instead of styling Link, I will prepare component that will be a button with link option (it will be possible to add `to` attr, etc) and we can use it with all buttons. 

I think it's a good idea with having all buttons in one component. What do you think?

Reply:
Sounds good to me